### DISCUSSION TITLE
Confused by controls in <View /> component

### DISCUSSION THREAD

**Discussion by janvarsa** [pmndrs/drei#863] [Q&A]
When I set up a Canvas with two View components and OrbitControls in one view and MapControls in the other, the controls in each View end up controlling both view simultaneously as in the example below:

https://codesandbox.io/s/shy-fog-9srrlp?file=/src/App.js

How do I make my controls reference only the View they were created in? The example below is nearly identical but behaves differently:

https://codesandbox.io/s/1wmlew?file=/src/App.js

**Comment by janvarsa** [comment]
I noticed this seems to work as expected if I downgrade to @react-three/fiber 8.0.10 but going to 8.0.11 breaks it.

**Reply by mwmwmw** [reply]
This is fixed in 8.0.12. If you upgrade the deps it works again.
